style/js/shield_defs.js

```diff
@@ -718,6 +718,51 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Loop"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["LOOP"]
+  );
+  shields["US:TX:Spur"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["SPUR"]
+  );
+  shields["US:TX:Business"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BUS"]
+  );
+  shields["US:TX:Park"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["PARK"]
+  );
+  shields["US:TX:Beltway"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BELT"]
+  );
+  shields["US:TX:FM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["F.M."]
+  );
+  shields["US:TX:RM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R.M."]
+  );
+  shields["US:TX:Recreational"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R"]
+  );
+  shields["US:TX:NASA"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["NASA"]
+  );
+
+  shields["US:TX:Toll"] = banneredShield(
+    roundedRectShield("#003f87", "white", "white", 1, 1, null),
+    ["TOLL"]
+  );
+  shields["US:TX:CTRMA"] = shields["US:TX:Toll"];
+  shields["US:TX:NTTA"] = shields["US:TX:Toll"];
+
```

Comment:
Existing `network` values for Texas county roads are:

- `US:TX:CR:Caldwell`
- `US:TX:CR:Collin`
- `US:TX:CR:Collingsworth`
- `US:TX:CR:Colorado`
- `US:TX:CR:Cooke`
- `US:TX:CR:Coryell`
- `US:TX:CR:Fannin`
- `US:TX:CR:Gonzales`
- `US:TX:CR:Guadalupe`
- `US:TX:CR:Hays`
- `US:TX:CR:Lamar`
- `US:TX:CR:Runnels`
- `US:TX:CR:San Patricio`
- `US:TX:CR:Smith`
- `US:TX:CR:Van Zandt`
- `US:TX:CR:Wilbarger`
- `US:TX:CR:Wilson`
- `US:TX:CR:Wood`
- `US:TX:CR:Wood:PR` (private roads)

AFAIK, county roads are signposted with either the standard blue pentagon or nothing at all. We haven't done the research yet on which of the 254 counties do and don't give their roads shields, so unless anybody has any insight on these specific counties, we could probably hold off on this.

Reply:
I agree with Minh.  County routes can be addressed in a separate PR.